README.md

```diff
@@ -232,6 +233,47 @@ dash.update(
 
 `Dashboard` updates will also update any `Chart` configurations it owns.
 
+    Note: If the given dashboard does not already exist, `update` will create a new dashboard for you
+
+<a name="dashboard-filters"></a>
+### Providing Dashboard Filters
+
+Dashboards can be configured to provide various filters that affect the behavior of all configured charts (overriding any conflicting filters at the chart level). You may wish to do this in order to quickly change the environment that you're observing for a given set of charts.
+
+
+```python
+from signal_analog.filters import DashboardFilters, FilterVariable
+app_var = FilterVariable().with_alias('app')\
+.with_property('app')\
```

Comment:
Will this example work with the current indentation levels on builder methods? i.e. do we need to indent things like `.with_property` so that the examples run correctly?